cmd/insprd/memory/tree/alias_test.go

```diff
@@ -388,18 +367,13 @@ func TestAliasMemoryManager_Delete(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			setTree(&MockManager{
-				MemoryManager: &MemoryManager{
-					root: tt.fields.root,
-					tree: tt.fields.root,
-				},
-				appErr: tt.fields.appErr,
-				mockC:  tt.fields.mockC,
-				mockA:  tt.fields.mockA,
-				mockCT: tt.fields.mockCT,
-			})
 
-			amm := GetTreeMemory().Alias()
+			mem := &treeMemoryManager{
```

Comment:
We've talked about Aliases never existing in the "MockManager" structure, but if you were able to run a test without the "MockManager" for the Alias, isn't it possible to do the same for dApps, Channels and Types? (It won't be as easy, but is it possible?)

Reply:
I think it is